sros2/api/__init__.py

```diff
@@ -128,21 +128,24 @@ def create_governance_file(path, domain_id):
         f.write("""\
 <?xml version="1.0" encoding="UTF-8"?>
 <dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="http://www.omg.org/spec/DDS-SECURITY/20140301/dds_security_governance.xsd">
+    xsi:noNamespaceSchemaLocation="http://www.omg.org/spec/DDS-SECURITY/20160303/omg_shared_ca_governance.xsd">
     <domain_access_rules>
         <domain_rule>
-            <domain_id>%s</domain_id>
-            <allow_unauthenticated_join>FALSE</allow_unauthenticated_join>
-            <enable_join_access_control>TRUE</enable_join_access_control>
+            <domains>
+              <id>%s</id>
+            </domains>
+            <!-- <allow_unauthenticated_join>false</allow_unauthenticated_join> SPECIFIED IN DDS SECURITY XSD -->
+            <allow_unauthenticated_participants>false</allow_unauthenticated_participants> <!-- SPECIFIED IN DDS SECURITY BUT NOT IN THE OFFICIAL XSD, IMPLEMENTED BY RTI DDSSEC-130 and RTI XSD http://community.rti.com/schema/5.3.0/dds_security_governance.xsd -->
```

Comment:
good point, fixed in https://github.com/ros2/sros2/pull/16/commits/e1e2c0bf1471888e8ff136242e3fec033c48ba6e